modules/release-notes/con-release-notes-overview.adoc

```diff
@@ -19,3 +19,22 @@ Benefits of {product} include:
 .Additional resources
 
 * link:{upgrading-book-url}[{upgrading-book-title}].
+
+== Supported Platforms
+
+{product} runs on OpenShift Container Platform 4.14-4.16 on the following CPU architecture:
+
+* AMD64 and Intel 64 (`x86_64`)
+
+In addition to OpenShift Container Platform, the following hosted environments are also supported:
+
+* {eks-brand-name}
+* {aks-brand-name}
+* {osd-short} on {gcp-brand-name}
+
+.Additional resources
+
+* link:https://docs.redhat.com/en/documentation/openshift_container_platform[OpenShift documentation]
+* link:https://docs.redhat.com/en/documentation/red_hat_developer_hub/1.3#Install%20and%20Upgrade[{product} Installation guides]
```

Comment:
I'm not certain about the link included here, as it may change in the future. It might be better to add the link to the product documentation instead.
```suggestion
* link:https://docs.redhat.com/en/documentation/red_hat_developer_hub/{product-version}#Install%20and%20Upgrade[{product} Installation guides]
```